user/darkhttpd/template.py

```diff
@@ -0,0 +1,19 @@
+pkgname = "darkhttpd"
+pkgver = "1.16"
+pkgrel = 0
+build_style = "makefile"
+make_use_env = True
+pkgdesc = "Single-threaded static content webserver"
+maintainer = "ttyyls <contact@behri.org>"
+license = "ISC"
+url = "https://unix4lyfe.org/darkhttpd"
+source = f"https://github.com/emikulic/darkhttpd/archive/v{pkgver}.tar.gz"
+sha256 = "ab97ea3404654af765f78282aa09cfe4226cb007d2fcc59fe1a475ba0fef1981"
+hardening = ["vis", "cfi"]
+# no tests defined
+options = ["!check"]
+
+
+def install(self):
+    self.install_license("COPYING")
```

Comment:
`COPYING` doesn't contain a copyright notice, so you can either e.g. import my patch from https://github.com/emikulic/darkhttpd/pull/70 or probably do this instead:
```suggestion
# check: no tests defined
# distlicense: license file doesn't include copyright notice, see
# https://github.com/emikulic/darkhttpd/pull/70
options = ["!check", "!distlicense"]


def install(self):
```

Reply:
the patch itself looks good and I suppose necessary for the license to be coherent. But a copyight notice is included in the executable itself and gets displayed first when run, as such Im not sure if it's necessary to block on it